### DISCUSSION TITLE
I can't compile in Nuxt 2

### DISCUSSION THREAD

**Discussion by msoler75** [CodeDredd/pinia-orm#834] [Q&A] [answered]
Hi. Thanks for this useful library. I'm trying to get run in Nuxt 2. But I have this compilation error:

```
This dependency was not found:                                                                                              
* #build/orm-options in ./node_modules/@pinia-orm/nuxt/dist/runtime/nuxt2-plugin.mjs 
```

When I check nuxt2-plugin.mjs, it contains:
`import { ormOptions } from "#build/orm-options";`

I put here some useful information of my setup:

nuxt.config.js:
```
 // Modules for dev and build (recommended): https://go.nuxtjs.dev/config-modules
  buildModules: [
    // https://go.nuxtjs.dev/eslint
    // '@nuxtjs/eslint-module'
    '@nuxtjs/composition-api/module',
    '@pinia/nuxt',
    '@pinia-orm/nuxt'
  ],
```

package.json:
```
 "dependencies": {
    "@nuxtjs/composition-api": "^0.33.1",
    "@nuxtjs/pwa": "^3.3.5",
    "@pinia-orm/nuxt": "^1.1.6",
    "@pinia/nuxt": "0.2.1",
    "core-js": "^3.25.3",
    "nuxt": "^2.15.8",
    "pinia": "^2.0.28",
    "pinia-orm": "^1.4.0",
    "vue": "^2.7.10",
    "vue-server-renderer": "^2.7.10",
    "vue-template-compiler": "^2.7.10"
  },
```

Any help would be appreciated

**Comment by CodeDredd** [comment, marked as answer]
Try version `1.0.18` for `@pinia-orm/nuxt`.
This new imports are only working with `bridge`. Didn't know that at some point i lost compo support, but thats the same with some pinia version.

**Reply by msoler75** [reply]
Thanks, I tried but it didn't work. It may be that an older version could work.

But for now, I decided to move my project to nuxt 3 for this plugin and other stuff to work.
